src/plugins/dashboard/public/application/embeddable/dashboard_container.tsx

```diff
@@ -186,7 +186,11 @@ export class DashboardContainer extends Container<InheritedChildInput, Dashboard
     ReactDOM.render(
       <I18nProvider>
         <KibanaContextProvider services={this.options}>
-          <DashboardViewport renderEmpty={this.renderEmpty} container={this} />
+          <DashboardViewport
+            renderEmpty={this.renderEmpty}
+            container={this}
+            PanelComponent={this.options.embeddable.EmbeddablePanel}
```

Comment:
This is a nice change. Could it go in a separate PR?  Seems unrelated to the focus of the state transfer.

Reply:
good point! I split all of the embeddable panel changes into: #68006